src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -132,22 +131,62 @@ public static Tag parseTag(String tag) throws ParseException {
     }
 
     /**
-     * Parses a {@code String dateTime} into an {@code LocalDateTime}.
+     * Parses a {@code String dateTime} into an {@code LocalDate}.
      * Leading and trailing whitespaces will be trimmed.
      *
-     * @param dateTime Date and time in String format of dd-MM-yyyy HH:mm.
-     * @return Parsed LocalDateTime representation of input.
-     * @throws ParseException if the given {@code dateTime} is invalid.
+     * @param dateTime Date in String format of dd-MM-yyyy HH:mm.
+     * @return Parsed LocalDate representation of input.
+     * @throws ParseException if the given {@code date} is invalid.
+     */
+    public static LocalDate parseAppointmentDate(String dateTime) throws ParseException {
+        requireNonNull(dateTime);
+        String trimmedDateTime = dateTime.trim();
+        String trimmedDate;
+
+        try {
+            trimmedDate = trimmedDateTime.split(" ")[0];
+        } catch (Exception e) {
+            throw new ParseException("Missing date/time!");
```

Comment:
Perhaps you can split the cases so that you can specify whether it is missing a date or time?